aries-site/src/components/feedback/Feedback.js

```diff
@@ -46,19 +43,24 @@ export const Feedback = ({
       >
         <Box {...theme?.feedback?.body}>
           <>{children}</>
-          <Box {...theme?.feedback?.footer}>{footerActions}</Box>
+          <Box {...theme?.feedback?.footer}>
+            {/* how would we decide which order? offer reverse prop? */}
+            {/* what about passing in other button props? primary icon? */}
```

Comment:
Maybe the buttons can be styled by the theme? the order can be handled by theme.feedback.footer.

It feels like for the first pass it's okay to be a bit prescriptive.

Reply:
okay cool but you like this direction of having a prop for the Buttons?

Reply:
It seems reasonable, but undecided. Since we are prototyping, we can try it out and see how it feels.

Reply:
yeah of course Im going to fix merge conflicts then merge it into the main branch